### DISCUSSION TITLE
Play 1 developers

### DISCUSSION THREAD

**Discussion by nikvouk-tickesq** [playframework/play1#1449] [General]
Hello,
  I have a play 1 project that needs some development and updating. I'm looking for any experienced Play 1 developers (java, jquery, javascript) to help me with development of the project on contract. Please contact me  privately or respond here if you are interested.
thank you,
Nikola

**Comment by PromanSEW** [comment]


**Reply by nvouk-godaddy** [reply]
Thanks for the suggestion. It seems play1 is only for issues, and this discussions forum is dedicated for all questions.

**Reply by PromanSEW** [reply]
Ah, I missed that discussions are for all organization, not for repository

**Reply by mkurz** [reply]
I enabled Discussions for the play1 repo now: https://github.com/playframework/play1/discussions
@nikvouk-tickesq you should ask your question(s) there.

**Reply by mkurz** [reply]
I moved the discussion to the play1 repo, I will now remove this comments

**Comment by teicher** [comment]
Excellent, thanks!

**Comment by xael-fry** [comment]
@nikvouk-tickesq So I think it is the good place to ask your questions :)
